GUI/Types/Renderer/GLSceneViewer.cs

```diff
@@ -87,7 +87,8 @@ private void OnLoad(object sender, EventArgs e)
             if (Scene.AllNodes.Any())
             {
                 var bbox = Scene.AllNodes.First().BoundingBox;
-                var location = new Vector3(bbox.Max.Z, 0, bbox.Max.Z) * 1.5f;
+                //if first node has no bbox, LookAt will break camera, so +1 to location.x
+                var location = new Vector3(bbox.Max.Z + 1, 0, bbox.Max.Z) * 1.5f;
```

Comment:
Shouldn't the second Z also need +1?

Reply:
Not really. We just need any non zero length vector